src/pages/nft/class/_classId/index.vue

```diff
@@ -440,6 +441,11 @@ export default {
     defaultSelectedValue() {
       return this.nftEditions[0]?.value;
     },
+    controlBarPriceLabel() {
+      return this.nftIsNFTBook
+        ? this.nftBookFeaturedPrice
+        : formatNumberWithLIKE(this.NFTPrice);
```

Comment:
I think a better way is to check if book store price exists, if not show LIKE price if exists.